frontend/app/templates/components/admin/user/table.hbs

```diff
@@ -1,12 +1,12 @@
 {{#if this.isUserNew}}
   <Admin::User::Form @onAbort={{this.toggleUserNew}} @onSuccess={{this.addUser}} @title={{t "admin.users.new.title"}} />
 {{/if}}
-<div class="container my-2">
-  <div class="row justify-content-end">
-    <div class="col-auto">
-      <button class="btn btn-primary" {{on "click" this.toggleUserNew}}>{{t "new"}}</button>
-    </div>
-  </div>
+<div class="row justify-content-end">
+      {{!-- <button class="btn btn-primary" {{on "click" this.toggleUserNew}}>{{t "new"}}</button> --}}
+        <span class="mx-2" role="button" {{on "click" this.toggleUserNew}}>
+          <img class="icon-big-button" src="/assets/images/plus.svg" alt="new">
+          <BsTooltip @title={{t "tooltips.add_folder"}} @delayShow="1000" />
+        </span>
```

Comment:
This looks somewhat worse like this no? The tooltip is not correct and maybe you could give it a margin from the right.
Right now it just looks a bit randomly placed without real context (at least to me)